discovery-provider/src/queries/query_helpers.py

```diff
@@ -786,7 +790,14 @@ def get_repost_counts(
     return repost_counts_query.all()
 
 
-def get_karma(session, ids, time=None, is_playlist=False, xf=False):
+def get_karma(
+    session: Session,
+    ids: List[int],
+    strategy: TrendingVersion,
+    time: str = None,
+    is_playlist: bool = False,
+    xf: bool = False,
+):
```

Comment:
Because we don't have a `trending_params` table which caches karma for for all of trending ie playlist & underground trending track, this was left here - no need to change now, but can eventually be direct queries to the cached karma used to calculate trending scores.